packages/shared/models/board.ts

```diff
@@ -0,0 +1,34 @@
+import mongoose from 'mongoose'
+
+const boardSchema = new mongoose.Schema({
+  id: String,
```

Comment:
change this field to _id use type: ObjectId
https://www.tutorialspoint.com/mongodb/mongodb_objectid.htm
to create test data you can use generator:
https://observablehq.com/@hugodf/mongodb-objectid-generator
eg. _id: ObjectId('6225239aa76ad13dd37178b6'),